S/SuiteSparse/SuiteSparse@Test/build_tarballs.jl

```diff
@@ -0,0 +1,118 @@
+include("../common.jl")
+
+name = "SuiteSparse_Test"
+version = v"7.1.1"
+
+using BinaryBuilder, Pkg
+
+# We enable experimental platforms as this is a core Julia dependency
+platforms = supported_platforms(;experimental=true)
+push!(platforms, Platform("x86_64", "linux"; sanitize="memory"))
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libsuitesparseconfig",   :libsuitesparseconfig),
+    LibraryProduct("libamd",                 :libamd),
+    LibraryProduct("libbtf",                 :libbtf),
+    LibraryProduct("libcamd",                :libcamd),
+    LibraryProduct("libccolamd",             :libccolamd),
+    LibraryProduct("libcolamd",              :libcolamd),
+    LibraryProduct("libcholmod",             :libcholmod),
+    LibraryProduct("libldl",                 :libldl),
+    LibraryProduct("libklu",                 :libklu),
+    LibraryProduct("libumfpack",             :libumfpack),
+    LibraryProduct("librbio",                :librbio),
+    LibraryProduct("libspqr",                :libspqr),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency("libblastrampoline_jll"; compat="5.4.0"),
+    BuildDependency("LLVMCompilerRT_jll",platforms=[Platform("x86_64", "linux"; sanitize="memory")]),
```

Comment:
As a general remark, ***never*** make comparison with exact platforms, it's finicky and error-prone, check that the sanitizer of the platform is `memory`: https://github.com/JuliaPackaging/Yggdrasil/blob/c291c43414edd807000867a7c38796dc266b3dd9/O/OpenBLAS/common.jl#L245
```suggestion
    BuildDependency("LLVMCompilerRT_jll"; platforms=filter(p -> sanitize(p)=="memory", platforms)),
```